assignments/pr_review_tutorial/ingram/README.md

```diff
@@ -1,11 +1,10 @@
-Ok so I guess you are reading this cuz you wanna use my code. There are some
-functions that do stuf and thats:
+Ok so I guess you are reading this cuz you wanna use my code. This code imports factorial from simple_functions
```

Comment:
I would suggest to use double backticks around function names

Reply:
Will do! 🙂